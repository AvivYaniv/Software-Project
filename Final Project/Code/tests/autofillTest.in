edit tests/b3_2_empty.txt
autofill
solve tests/b3_2_empty.txt
set 1 1 1
set 4 4 4
autofill
undo
undo
undo
redo
print_board
redo
print_board
solve tests/b2_2.txt
autofill
autofill
autofill
undo
print_board
redo
print_board
set 1 2 4
autofill
autofill
set 3 4 1
autofill
undo
autofill
exit