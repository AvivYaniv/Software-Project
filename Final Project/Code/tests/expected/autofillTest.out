Sudoku
------
Enter your command:
----------------------------
|        |        |        |
|        |        |        |
|        |        |        |
----------------------------
|        |        |        |
|        |        |        |
|        |        |        |
----------------------------
Enter your command:
ERROR: invalid command
Enter your command:
----------------------------
|        |        |        |
|        |        |        |
|        |        |        |
----------------------------
|        |        |        |
|        |        |        |
|        |        |        |
----------------------------
Enter your command:
----------------------------
|  1     |        |        |
|        |        |        |
|        |        |        |
----------------------------
|        |        |        |
|        |        |        |
|        |        |        |
----------------------------
Enter your command:
----------------------------
|  1     |        |        |
|        |        |        |
|        |        |        |
----------------------------
|        |      4 |        |
|        |        |        |
|        |        |        |
----------------------------
Enter your command:
----------------------------
|  1     |        |        |
|        |        |        |
|        |        |        |
----------------------------
|        |      4 |        |
|        |        |        |
|        |        |        |
----------------------------
Enter your command:
----------------------------
|  1     |        |        |
|        |        |        |
|        |        |        |
----------------------------
|        |        |        |
|        |        |        |
|        |        |        |
----------------------------
Undo 4,4: from 4 to _
Enter your command:
----------------------------
|        |        |        |
|        |        |        |
|        |        |        |
----------------------------
|        |        |        |
|        |        |        |
|        |        |        |
----------------------------
Undo 1,1: from 1 to _
Enter your command:
Error: no moves to undo
Enter your command:
----------------------------
|  1     |        |        |
|        |        |        |
|        |        |        |
----------------------------
|        |        |        |
|        |        |        |
|        |        |        |
----------------------------
Redo 1,1: from _ to 1
Enter your command:
----------------------------
|  1     |        |        |
|        |        |        |
|        |        |        |
----------------------------
|        |        |        |
|        |        |        |
|        |        |        |
----------------------------
Enter your command:
----------------------------
|  1     |        |        |
|        |        |        |
|        |        |        |
----------------------------
|        |      4 |        |
|        |        |        |
|        |        |        |
----------------------------
Redo 4,4: from _ to 4
Enter your command:
----------------------------
|  1     |        |        |
|        |        |        |
|        |        |        |
----------------------------
|        |      4 |        |
|        |        |        |
|        |        |        |
----------------------------
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
Cell <4,1> set to 3
-------------------
|  1.  2.|  4.  3 |
|        |      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
Cell <3,2> set to 2
Cell <4,4> set to 4
-------------------
|  1.  2.|  4.  3 |
|        |  2   1.|
-------------------
|        |      2.|
|        |      4 |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|        |  2   1.|
-------------------
|        |      2.|
|        |      4 |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|        |      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Undo 3,2: from 2 to _
Undo 4,4: from 4 to _
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|        |      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|        |  2   1.|
-------------------
|        |      2.|
|        |      4 |
-------------------
Redo 3,2: from _ to 2
Redo 4,4: from _ to 4
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|        |  2   1.|
-------------------
|        |      2.|
|        |      4 |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|  4     |  2   1.|
-------------------
|        |      2.|
|        |      4 |
-------------------
Enter your command:
Cell <2,2> set to 3
Cell <1,3> set to 3
-------------------
|  1.  2.|  4.  3 |
|  4   3 |  2   1.|
-------------------
|  3     |      2.|
|        |      4 |
-------------------
Enter your command:
Cell <3,3> set to 1
Cell <1,4> set to 2
Cell <2,4> set to 1
-------------------
|  1.  2.|  4.  3 |
|  4   3 |  2   1.|
-------------------
|  3     |  1   2.|
|  2   1 |      4 |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|  4   3 |  2   1.|
-------------------
|  3     |  1*  2.|
|  2   1*|  1*  4 |
-------------------
Enter your command:
Error: board contains erroneous values
Enter your command:
-------------------
|  1.  2.|  4.  3 |
|  4   3 |  2   1.|
-------------------
|  3     |  1   2.|
|  2   1 |      4 |
-------------------
Undo 3,4: from 1 to _
Enter your command:
Cell <2,3> set to 4
Cell <3,4> set to 3
-------------------
|  1.  2.|  4.  3 |
|  4   3 |  2   1.|
-------------------
|  3   4 |  1   2.|
|  2   1 |  3   4 |
-------------------
Puzzle solved successfully
Enter your command:
Exiting...
